database/enums/enum_enrollment_status.pg

```diff
@@ -0,0 +1 @@
+invited, joined, removed, blocked
```

Comment:
Not an actionable comment, just lamenting the fact that we're already quite inconsistent in how our enums are cased, there's a mixture of `snake_case` and `PascalCase`.